tasks/build.py

```diff
@@ -99,8 +101,10 @@ def get_architecturetargets():
         dict(str, dict): dictionary of arch_target_map which contains entries of the format
                          OS/SUBDIR : ADDITIONAL_SBATCH_PARAMETERS
     """
-    architecturetargets = config.get_section('architecturetargets')
-    arch_target_map = json.loads(architecturetargets.get('arch_target_map'))
+    cfg = config.read_config('app.cfg')
+    architecturetargets_cfg = cfg['buildenv']
+    architecturetargets = architecturetargets_cfg['architecturetargets']
```

Comment:
This line can be removed, since it's not used at all...

There's actually no such setting at all, so without using `.get` this will trigger a hard crash!